org.contour.Terminal.yml

```diff
@@ -0,0 +1,92 @@
+app-id: org.contour.Terminal
```

Comment:
`contour.org` redirect to forums about the Ford Contour. Is this a really a valid domain the developers control?

It doesn't look like it.

Reply:
This is what I was trying to argue about in https://github.com/flathub/flathub/pull/3349#issuecomment-1193103621

I own the domain `contour-terminal.org`, which is forwarding to github pages (not yet set up => 404).

However, `contour-terminal` being part of the app id seems invalid, flatpak-builder was at least bailing out saying that. So I have to chose something I do not own, because it does not allow me to specify of what I own. When reading the docs, it did not seem like a hard requirement, so this is why i was giving this explanation above. :)

If you have better proposals for the App ID, I am all open :-)